ui/src/components/SampleControls/SampleControls.jsx

```diff
@@ -1,95 +1,35 @@
 import React from 'react';
-import { Dropdown } from 'react-bootstrap';
 
 import SnapshotControl from './SnapshotControl';
-
-import styles from './SampleControls.module.css';
 import GridControl from './GridControl';
 import CentringControl from './CentringControl';
 import FocusControl from './FocusControl';
 import ZoomControl from './ZoomControl';
 import LightControl from './LightControl';
+import VideoSizeControl from './VideoSizeControl';
 
-class SampleControls extends React.Component {
-  constructor(props) {
-    super(props);
-    this.availableVideoSizes = this.availableVideoSizes.bind(this);
-  }
-
-  availableVideoSizes() {
-    const items = this.props.videoSizes.map((size) => {
-      const sizeGClass =
-        this.props.width === String(size[0])
-          ? 'fas fa-circle'
-          : 'far fa-circle';
-
-      return (
-        <Dropdown.Item
-          key={`${size[0]} x ${size[1]}`}
-          eventKey="1"
-          onClick={() =>
-            this.props.sampleViewActions.setVideoSize(size[0], size[1])
-          }
-        >
-          <span className={sizeGClass} /> {`${size[0]} x ${size[1]}`}
-        </Dropdown.Item>
-      );
-    });
-
-    items.push(
-      <Dropdown.Item
-        eventKey="3"
-        key="reset"
-        onClick={() => {
-          this.props.sampleViewActions.setVideoSize(
-            this.props.width,
-            this.props.height,
-          );
-        }}
-      >
-        <span className="far fa-redo" /> Reset
-      </Dropdown.Item>,
```

Comment:
I've removed the _Reset_ item from the _Video size_ dropdown, as I don't think it did anything (it just reapplies the current width/height...) Did it ever work? If so what was its purpose?

Reply:
I think its a regression from the past, It used to change to set the video size to the current size, meaning that it would simply restart the streaming with the same size

Reply:
It was working as expected then, but two questions then: 
- Why would one want to restart de streaming ? Can it ever stop? 
- Would a dedicated button called "restart streaming" somewhere be clearer?

Reply:
As discussed, re-selecting the current video size has the same effect as the now-removed "Reset" button, so I won't re-add the "Reset" button.

Reply:
Thanks !